onecodex/cli.py

```diff
@@ -60,6 +62,11 @@ def onecodex(ctx, api_key, no_pprint, verbose, telemetry):
     if verbose:
         log.setLevel(logging.INFO)
 
+    # Show a warning if simplejson is installed
```

Comment:
Let's just open an issue to track fixing this upstream so we can remove this annoying warning (i.e., to make it work with `simplejson`).